ci-templates/github/diagram-cache/action.yml

```diff
@@ -0,0 +1,67 @@
+# SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+# SPDX-License-Identifier: Apache-2.0
+
+name: 'Generate diagram cache'
+description: 'An action to create and push a capella diagram cache'
+
+inputs:
+  commit_msg:
+    description: 'The commit message for updating the diagram cache'
+    required: false
+    default: "docs: Update diagram cache"
+  entry_point:
+    description: 'The entry point of the used model'
+  capellambse_revision:
+    description: 'The revision of the py-capellambse package to install'
+    required: false
+    default: master
+  capella_docker_images_revision:
+    description: 'The revision of capella-dockerimages to use'
+    required: false
+    default:  main
+  capella_docker_images_dropins:
+    description: 'Please have a look at https://dsd-dbs.github.io/capella-dockerimages/capella/introduction/#supported-dropins'
+    required: false
+    default: selected
+  push_diagram_cache:
+    description: 'Whether to push the diagram cache changes or not'
+    required: false
+    default: "true"
+  commit_branch:
+    description: 'The base branch to which the diagram cache belongs'
```

Comment:
I'd remove this input and don't make it configurable.

Reply:
done